tests/epyccel/test_arrays.py

```diff
@@ -3975,11 +3970,51 @@ def test_array_ndmin_2_order(language):
     check_array_equal(f1(d), f2(d))
     check_array_equal(f1(e), f2(e))
 
+##==============================================================================
+## TEST ITERATION
+##==============================================================================
+
+def test_iterate_slice(language):
+    f1 = arrays.iterate_slice
+    f2 = epyccel(f1, language = language)
+    i = randint(2, 10)
+    assert f1(i) == f2(i)
 
-#def teardown_module():
-#    import os, glob
-#    dirname  = os.path.dirname( arrays.__file__ )
-#    pattern  = os.path.join( dirname, '__epyccel__*' )
-#    filelist = glob.glob( pattern )
-#    for f in filelist:
-#        os.remove( f )
+def test_unpacking(language):
+    f1 = arrays.unpack_array
+    f2 = epyccel(f1, language = language)
+
+    arr = np.arange(3, dtype=int)
+    assert f1(arr) == f2(arr)
+
+    arr = np.arange(12, dtype=int).reshape((3,4))
+    x1, y1, z1 = f1(arr)
+    x2, y2, z2 = f1(arr)
```

Comment:
Should this not read the following?
```suggestion
    x1, y1, z1 = f1(arr)
    x2, y2, z2 = f2(arr)
```

Reply:
Fixed 7109367dba8f448047fca9f0c488101fe6c178ee